sdk/communication/Azure.Communication.CallingServer/src/Models/CallAutomationModelFactory.cs

```diff
@@ -120,23 +120,6 @@ public static AddParticipantsSucceeded AddParticipantsSucceeded(string operation
             return new AddParticipantsSucceeded(internalObject);
         }
 
-        /// <summary>
```

Comment:
Are we removing CallConnected event?

Reply:
There is a generated CallAutomationModelFactory that already has this method. It actually fails saying that the call is ambiguous.

Reply:
Actually, I don't know if we should be able to delete the entire ModelFactory... there is not enough testing covering all the events to tell.